File: python/array/nestedArray.py
nested = [1, [2, [3, 4]]]

# flatten manual
flat = []
for item in nested:
    if isinstance(item, list):
        flat.extend(item)
    else:
        flat.append(item)

# flatten cepat (1 level)
import itertools
flat = list(itertools.chain.from_iterable([[1,2],[3,4]]))

File: javascript/array/nestedArray.js
const nested = [1, [2, [3, 4]]];

// flat
nested.flat(2);          // [1, 2, 3, 4]

// flatMap
[1, 2, 3].flatMap(x => [x, x * 2]);
// [1, 2, 2, 4, 3, 6]